.gitignore

```diff
@@ -6,6 +6,7 @@
 
 # Ignore bundler config.
 /.bundle
+/spec/cassettes/*
```

Comment:
I haven't been doing this since I put the tokens in the application.yml file, but let's just plan on keeping these off github!